apps/web/modules/email/components/preview-email-template.tsx

```diff
@@ -54,29 +55,26 @@ export async function PreviewEmailTemplate({
   const urlWithPrefilling = `${surveyUrl}?preview=true&skipPrefilled=true&`;
   const defaultLanguageCode = "default";
   const firstQuestion = survey.questions[0];
-
+  const headline = parseRecallInfo(getLocalizedValue(firstQuestion.headline, defaultLanguageCode), {}, {});
```

Comment:
do we need the empty objects here? I think since they are optional and we use the `Object.keys` function, they will get evaluated to an empty array which is fine in itself